pkg/convert/pot.go

```diff
@@ -40,10 +48,22 @@ func ToPot(t model.Translation) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+func writePluralHeader(b *bytes.Buffer, t model.Translation) error {
+	for _, message := range t.Messages {
+		if message.PluralID != "" {
+			if _, err := fmt.Fprintf(b, "\"Plural-Forms: nplurals=%d; plural=(n != 1);\\n\"\n\n", pluralCountLimit); err != nil { //nolint:lll
+				return fmt.Errorf("write plural forms: %w", err)
+			}
+
+			break
```

Comment:
`return` conveys the meaning and intent better. If you use `break`, it means there is additional processing but there is none actually.

```suggestion
			return nil
```